aws/rust-runtime/aws-types/src/credentials/provider.rs

```diff
@@ -44,7 +44,7 @@ impl Display for CredentialsError {
     fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
         match self {
             CredentialsError::CredentialsNotLoaded => {
-                write!(f, "The provider could not provide credentials or required configuration was not set")
+                write!(f, "The credential provider was not enabled")
```

Comment:
yeah, what I think we need here is to add a context field, (#767) so that providers can provide a bit more info about what's going on (eg. we tried all the providers in the default chain but we didn't find anything, etc.)

But that's a bit more than I want in this PR